### PR TITLE
check if workflow has started

### PR DESCRIPTION
**Pull request by KPostOffice**
## Related Issues and Dependencies

https://github.com/thoth-station/common/issues/1014

## This introduces a breaking change

- [ ] Yes
- [x] No

## This Pull Request implements

argo api changed slightly

### CODE REVIEW

File: thoth/common/workflows.py [hunk 1]
```diff
@@ -462,11 +462,12 @@ def submit_workflow(
 
     def get_pending_workflows(self, workflow_namespace: str) -> int:
         """Get the total number of workflows in a given namespace."""
```
Comment (pacospace) [inline thoth/common/workflows.py:464]: total number of pending workflows?

**Review by pacospace — COMMENTED** [review]
One minor comment otherwise LGTM

File: thoth/common/workflows.py [hunk 2]
```diff
@@ -461,12 +461,13 @@ def submit_workflow(
...
     def get_pending_workflows(self, workflow_namespace: str) -> int:
-        """Get the total number of workflows in a given namespace."""
-        return len(
-            self.api.list_namespaced_workflows(
-                workflow_namespace, labels="workflows.argoproj.io/phase=Pending"
-            ).items
-        )
+        """Get the total number of pending workflows in a given namespace."""
+        count = 0
+        self.api.list_namespaced_workflows(workflow_namespace,)
```
Comment (pacospace) [inline thoth/common/workflows.py:466]: is this line required?

Comment (KPostOffice) [reply thoth/common/workflows.py:466]: Ahh, no. My bad

**Review by pacospace — APPROVED** [review]
Thanks!

**Pull request merged**